src/app/layouts/SimplePageLayout.tsx

```diff
@@ -14,61 +15,70 @@ const SimplePageLayout: React.FC<SimplePageLayoutProps> = ({
   const appEnv = useAppEnv();
 
   return (
-    <ContentContainer
-      className={classNames(
-        "min-h-screen",
-        "flex flex-col",
-        !appEnv.fullPage && "bg-primary-white"
+    <>
+      {!appEnv.fullPage && (
+        <OverscrollBg
+          topClassName="bg-primary-white"
+          bottomClassName="bg-white"
+        />
       )}
-    >
-      <div
+
+      <ContentContainer
         className={classNames(
-          "flex-1",
-          "flex flex-col items-center justify-center"
+          "min-h-screen",
+          "flex flex-col",
+          !appEnv.fullPage && "bg-primary-white"
         )}
       >
-        <div className="mb-4 flex items-center text-gray-700">
-          <img src="../misc/icon.png" alt="" width="40" height="40" />
-
-          <span className="font-semibold ml-1 text-2xl tracking-tight">
-            Thanos
-          </span>
-        </div>
-
-        <h1 className="text-4xl text-gray-700 font-light">{title}</h1>
-      </div>
-
-      {appEnv.fullPage ? (
         <div
           className={classNames(
-            "w-full mx-auto max-w-md",
-            "bg-white",
-            "rounded-md shadow-md",
-            "px-4"
+            "flex-1",
+            "flex flex-col items-center justify-center"
           )}
         >
-          {children}
+          <div className="mb-4 flex items-center text-gray-700">
+            <img src="../misc/icon.png" alt="" width="40" height="40" />
+
+            <span className="font-semibold ml-1 text-2xl tracking-tight">
+              Thanos
+            </span>
+          </div>
+
+          <h1 className="text-4xl text-gray-700 font-light">{title}</h1>
         </div>
-      ) : (
+
+        {appEnv.fullPage ? (
```

Comment:
It can be potentially shorter: 

```tsx
<div
      className={classNames(
        (appEnv.fullPage
          ? [
              "w-full mx-auto max-w-md",
              "bg-white",
              "rounded-md shadow-md",
              "px-4"
            ]
          : ["-mx-4 px-4", "bg-white", "border-t border-gray-200", "shadow-md"])
      )}
    >
      {children}
    </div>
```